tests/sentry/integrations/jira_server/test_integration.py

```diff
@@ -1,433 +1,834 @@
-import responses
-from django.test.utils import override_settings
-from requests.exceptions import ReadTimeout
+import copy
+from unittest import mock
+from unittest.mock import patch
 
-from sentry.integrations.jira_server import JiraServerIntegrationProvider
-from sentry.models import Identity, IdentityProvider, Integration, OrganizationIntegration
-from sentry.testutils import IntegrationTestCase
-from sentry.utils import json, jwt
+import pytest
+import responses
+from django.urls import reverse
+
+from fixtures.integrations import StubService
+from fixtures.integrations.jira import StubJiraApiClient
+from sentry.models import (
+    ExternalIssue,
+    Integration,
+    IntegrationExternalProject,
+    OrganizationIntegration,
+)
+from sentry.shared_integrations.exceptions import IntegrationError
+from sentry.testutils import APITestCase
+from sentry.testutils.factories import DEFAULT_EVENT_DATA
+from sentry.testutils.helpers.datetime import before_now, iso_format
+from sentry.utils import json
+from sentry.utils.http import absolute_uri
+
+from . import get_integration
+
+
+def get_client():
+    return StubJiraApiClient()
+
+
+class JiraServerIntegrationTest(APITestCase):
```

Comment:
This is almost all the same as the tests from Jira Cloud's `test_integration.py`